tests/unit/s2n_config_test.c

```diff
@@ -731,5 +732,120 @@ int main(int argc, char **argv)
         EXPECT_SUCCESS(s2n_connection_set_config(conn, config));
     };
 
+    /* Test loading system certs */
+    {
+        /* s2n_config_load_system_certs safety */
+        {
+            EXPECT_FAILURE_WITH_ERRNO(s2n_config_load_system_certs(NULL), S2N_ERR_NULL);
+        }
+
+        /* s2n_minimal_config_new should not load system certs */
+        {
+            DEFER_CLEANUP(struct s2n_config *config = s2n_minimal_config_new(), s2n_config_ptr_free);
+            EXPECT_NOT_NULL(config);
+            EXPECT_NULL(config->trust_store.trust_store);
```

Comment:
Is it a problem during the handshake if the trust_store is NULL?

Reply:
Looks like s2n_x509_trust_store_wipe already set it to NULL, but I'd still maybe double check we can handle a null trust_store :)

Reply:
Yeah, `s2n_config_wipe_trust_store` also sets the trust store to NULL, so I think this is okay. I added a couple of self-talk tests to make sure handshakes are still behaving properly, though.